starlite/config/app.py

```diff
@@ -116,67 +119,67 @@ class Config(BaseConfig):
     """If set this enables the builtin CORS middleware."""
     csrf_config: Optional[CSRFConfig]
     """If set this enables the builtin CSRF middleware."""
-    debug: bool
+    debug: bool = True
     """If ``True``, app errors rendered as HTML with a stack trace."""
-    dependencies: Dict[str, Provide]
+    dependencies: Dict[str, Provide] = {}
     """A string keyed dictionary of dependency :class:`Provider <starlite.datastructures.Provide>` instances."""
     etag: Optional[ETag]
     """An ``etag`` header of type :class:`ETag <starlite.datastructures.ETag>` to add to route handlers of this app.
 
     Can be overridden by route handlers.
     """
-    event_emitter_backend: Type[BaseEventEmitterBackend]
+    event_emitter_backend: Type[BaseEventEmitterBackend] = SimpleEventEmitter
     """A subclass of :class:`BaseEventEmitterBackend <starlite.events.emitter.BaseEventEmitterBackend>`."""
-    exception_handlers: ExceptionHandlersMap
+    exception_handlers: Optional[ExceptionHandlersMap] = None
     """A dictionary that maps handler functions to status codes and/or exception types."""
-    guards: List[Guard]
+    guards: List[Guard] = []
     """A list of :class:`Guard <starlite.types.Guard>` callables."""
-    initial_state: InitialStateType
+    initial_state: Optional[InitialStateType] = None
     """An object from which to initialize the app state."""
-    listeners: List[EventListener]
+    listeners: List[EventListener] = []
     """A list of :class:`EventListener <starlite.events.listener.EventListener>`."""
-    logging_config: Optional[BaseLoggingConfig]
+    logging_config: Union["BaseLoggingConfig", "EmptyType", None]
```

Comment:
```suggestion
    logging_config: Optional[Union["BaseLoggingConfig", "EmptyType"]]
```

Reply:
I don't believe this isn't intended to be allowed to be both union `Empty` _and_ `Optional` on the config object. The possible `Empty` value is resolved away [here](https://github.com/starlite-api/starlite/blob/4cc867c0c5f49449b64932a391029b41b295b14c/starlite/app.py#L304).